src/env.ts

```diff
@@ -133,5 +131,12 @@ export function getEnv(
     }
   }
 
-  return getDefaultEnv(systemEnvVariables)
+  const envVars =
+    typeof systemEnvVariables === 'undefined' &&
+    typeof process === 'object' &&
+    typeof process.env === 'object'
+      ? process.env
+      : systemEnvVariables
```

Comment:
This should keep the same behaviour for projects relying on the `process.env` and passing no param to the `getEnv` helper.

If no param is sent to the helper, and `process.env` exists, it uses that as the `envVars`, otherwise if something custom is passed as param, it will use the custom param.

All the existing tests are working without having to make any changes on the tests themselves.